### DISCUSSION TITLE
Tutorial Resources

### DISCUSSION THREAD

**Discussion by wpcreative-jw** [flyntwp/flynt#425] [Q&A] [answered]
Does anyone know of any video resources that show:

1. Getting the theme setup and working
2. Actually building out a custom theme using flynt

Thanks

**Comment by aaronmeder** [comment, marked as answer]
Nope, but I thought about making such a video series for a while. 

Sadly too busy at the moment, but would love to to it since Flynt is the best WP setup I've had in years...love to work with it! ❤️

**Reply by purchasingtoolpak** [reply]
Not any I found specifically on Flynt. Although there are some good videos on Timber / Twig - which would help get you started understanding how Flynt works. Check this one out.
https://www.youtube.com/watch?v=Toq3CXgwWgw
